src/gui/include/gui/gui.h

```diff
@@ -153,6 +153,10 @@ class Painter
   };
   virtual void setBrush(const Color& color, const Brush& style = SOLID) = 0;
 
+  // Set the pen to an RGBA value and the brush
+  virtual void setPenAndBrush(const Color& color, bool cosmetic = false, const Brush& style = SOLID)
+      = 0;
```

Comment:
Is there a reason someone would override this with anything other than the obvious?  It seems more like a utility method that doesn't need to be virtual.